tests/data-files/S2A_MSIL1C_20210908T042701_N0301_R133_T46RER_20210908T070248.SAFE/expected_output.json

```diff
@@ -100,7 +100,7 @@
     },
     {
       "rel": "self",
-      "href": "/var/folders/3q/jbg6x0zx3194zq6_2jbwygjw0000gn/T/tmpkytdhlgs/S2A_MSIL1C_20210908T042701_R133_T46RER_20210908T070248.json",
+      "href": "/var/folders/3q/jbg6x0zx3194zq6_2jbwygjw0000gn/T/tmpwn6zz4bw/S2A_MSIL1C_20210908T042701_R133_T46RER_20210908T070248.json",
```

Comment:
For these expected output files, I find it can help to:
- remove all self links
- use relative asset links

e.g. https://github.com/stactools-packages/modis/blob/main/scripts/create_expected.py

Reply:
cool, i'll look at fixing that.